components/browser/search/build.gradle

```diff
@@ -40,7 +40,7 @@ dependencies {
 
     testImplementation project(':support-test')
 
-    testImplementation Dependencies.kotlin_coroutines_test
+    testImplementation Dependencies.testing_coroutines
```

Comment:
nitpick: Could you move this line so its in alphabetical order? (Between `testImplementation Dependencies.androidx_test_junit` and `testImplementation Dependencies.testing_robolectric`)

Reply:
Done.